js/deribit.js

```diff
@@ -133,7 +133,7 @@ module.exports = class deribit extends Exchange {
                         'get_email_language',
                         'get_new_announcements',
                         'get_position',
-                        'get_positions',
+                        'positions',
```

Comment:
They say it should be `get_positions` in their docs here:
- https://docs.deribit.com/#private-get_positions
Can you share some details on it?

Reply:
@dmskr can you plz share the source of the info (why get_positions → positions) ?

Reply:
Looks like you're right and original code uses correct API (despite weird resulting name, like privateGetGetPositions) with currency required.
After the change it seem to call this endpoint: https://test.deribit.com/pages/docs/api#section-24 and does not require currency. All positions returned by default (method name looks like: privateGetPositions). Retested this on production and both calls are working despite not documented in one or another version of the docs.
Still not sure if that's an experimental or deribit team just forgot to describe the changes they do. Anyway get positions without currency required is significantly more convenient method because of options and tons of possible combinations of instruments per account.

Reply:
@dmskr ok, thanks so much for the feedback! I would stick to the documented endpoint, and I'd close this PR for now if you don't mind. Let me know if I'm still missing something.

Reply:
Ah, pardon, I see `get_positions` along with `get_position`, so I'll merge it shortly.

Reply:
Since the api the change is all about is not documented and probably not official, probably, it makes sense to close the PR or leave it as is until reason for the difference will be clarified. So, yes, im ok with either outcome.

Reply:
@dmskr if i recall correctly, the `positions` endpoint is related to the older version of their API. Closing this for now, feel free to reopen it or just ask further questions if any. Thanks again for your involvement! We will be happy to see more PRs from you in the future!